src/Datadog.Trace/RuntimeMetrics/PerformanceCounterWrapper.cs

```diff
@@ -60,7 +60,7 @@ private void RefreshCounter(string instanceName)
             }
             catch (Exception ex)
             {
-                Log.Information(ex, "Error while renewing counter");
+                Log.Warning(ex, "Error while renewing counter");
```

Comment:
I think this one should be left to a lower level. If a counter has some kind of issues, since it's polled every 10 seconds, we'll still end up with a message every minute which will add significant noise.

Maybe we can (in a separate PR) add specific logic for that part, to log a message when it fails the first time, and stop logging until it starts working again

Reply:
Yep, makes sense 👍

Reply:
Actually, the default log level is `Information`, so this is already the case. In fact, we'll now be logging _less_ than before (thanks to rate limiting). `Warning` seems to more accurately describe the message, so maybe best to leave it?

Reply:
Alright